lib/garbage_collector.rb

```diff
@@ -24,11 +24,19 @@ def initialize(bintray_user:, bintray_key:, verbose: false, dry_run: false)
   #
   # @param projects [Array] the array of +Hash+ projects on which to run the garbage collection
   def run(projects)
+    projects_not_found = []
+
     projects.each do |project|
       log "Looking for versions to delete for #{project[:bintray]}..."
 
       dev_versions = @bintray.get_bintray_versions(project: project[:bintray])
 
+      if dev_versions.nil?
+        log "> Project not found."
+        projects_not_found.push(project)
+        next
+      end
```

Comment:
I made a typo in the project name and I discovered this bug. If we don't exit here, the process will try to call a method on `nil` and fail.